src/EnergyPlus/HeatBalanceKivaManager.cc

```diff
@@ -1067,7 +1067,7 @@ void KivaManager::calcKivaInstances()
 	for ( auto& kv : kivaInstances ) {
 		auto& grnd = kv.ground;
 		kv.setBoundaryConditions();
-		grnd.calculate( kv.bcs,DataGlobals::MinutesPerTimeStep*60. );
+		grnd.calculate( kv.bcs, timestep);
```

Comment:
@nealkruis If I understand correctly, Kiva was always running with the E+ timestep even if "hourly" was specified?

Reply:
@shorowit the calculation was still happening every hour, but the duration used to calculate the dynamics in the finite difference calculation was using the E+ timestep. Does that make sense?

For the current release (8.8), every model should specify "Timestep" for greatest accuracy.